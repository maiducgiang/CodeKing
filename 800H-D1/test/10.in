10
42
12 6 41 4 33 35 31 1 26 39 22 11 7 29 11 20 1 16 34 6 12 33 6 2 16 35 39 39 12 14 39 29 13 22 34 26 37 12 28 18 7 37 
60
35 21 26 19 17 39 27 19 26 20 21 29 60 19 27 57 51 16 41 27 39 7 58 53 16 6 6 51 34 42 18 37 14 47 33 4 52 48 21 57 25 19 32 25 3 18 33 1 24 3 13 26 6 10 23 40 43 6 57 43 
43
13 13 12 15 29 42 12 27 11 28 13 35 39 11 40 21 40 20 7 21 29 41 1 15 12 40 15 43 8 36 1 43 24 25 15 21 27 16 7 22 37 11 14 
57
15 11 30 57 5 18 40 44 41 3 56 42 9 45 18 37 22 53 54 38 49 20 42 21 27 57 30 7 50 35 10 31 37 1 4 41 21 4 7 47 55 34 43 32 26 44 12 49 7 35 44 52 48 24 12 49 40 
65
27 45 41 52 54 26 45 21 4 40 62 28 31 56 60 8 50 20 18 51 4 34 31 9 62 60 64 16 30 18 10 47 29 51 61 65 45 51 59 35 60 65 43 13 49 53 17 52 30 42 47 7 9 50 27 50 9 11 49 36 17 64 32 52 29 
50
11 29 21 39 32 32 8 34 26 43 45 43 20 39 23 46 30 33 14 28 5 43 33 23 17 43 28 2 19 2 25 49 15 21 50 7 7 29 14 24 41 25 36 11 16 11 25 47 43 22 
43
7 16 36 8 27 34 20 32 37 40 40 42 10 1 5 31 38 40 39 37 38 35 11 21 6 19 31 34 27 21 6 32 42 41 41 15 16 43 11 38 14 17 25 
66
21 63 63 30 46 19 20 49 55 4 48 16 20 16 44 46 9 55 10 44 31 38 34 63 52 35 17 65 47 20 46 60 41 46 48 2 40 34 42 50 9 37 17 2 14 47 1 16 19 66 52 1 66 22 31 4 31 32 28 31 24 21 42 34 25 17 
69
64 58 10 6 8 56 12 10 69 60 11 23 10 46 2 23 38 27 62 36 36 41 12 31 2 63 41 50 1 12 1 44 58 4 23 12 48 41 2 11 37 58 54 33 22 7 12 22 31 68 2 22 65 11 48 19 28 3 14 38 12 59 4 45 17 69 9 46 24 
61
45 47 33 47 23 51 45 28 4 51 53 23 33 37 24 22 19 27 14 23 25 8 8 7 48 3 23 50 39 25 9 41 3 18 12 56 13 8 20 37 5 34 17 55 10 24 20 54 20 25 38 13 1 7 42 45 10 26 4 61 37 
