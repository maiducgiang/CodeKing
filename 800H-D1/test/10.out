17
26
13
32
32
17
20
15
35
9
